README

```diff
@@ -2,10 +2,10 @@ Magpie
 ------
 
 Magpie contains a number of scripts for running Big Data software in
-HPC environments.  Thus far, Hadoop, Spark, Hbase, Storm, Pig, Mahout,
-Phoenix, Tachyon, and Zookeeper are supported.  It currently supports
-running over the parallel file system Lustre and running over any
-generic network filesytem.  There is scheduler/resource manager
+HPC environments.  Thus far, Hadoop, Spark, Kafka, Hbase, Storm, Pig, 
+Mahout, Phoenix, Tachyon, and Zookeeper are supported.  It currently 
+supports running over the parallel file system Lustre and running over
+any generic network filesytem.  There is scheduler/resource manager
 support for Slurm, Moab, Torque, and LSF.
```

Comment:
This is a personal nit, but could we put Kafka later in the list, like maybe after Phoenix?  I like to keep Hadoop, Spark, & Hbase first b/c they are the bigger name projects that people like/see.